webapp/locales/en.json

```diff
@@ -199,7 +199,11 @@
   "search": {
     "placeholder": "Search",
     "hint": "What are you searching for?",
-    "failed": "Nothing found"
+    "failed": "Nothing found",
+    "heading": {
+      "Post": "Posts",
+      "User": "Users"
+    }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-07T11:30:40Z" title="Tuesday, January 7th 2020, 12:30:40 pm +01:00">Jan 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -199,7 +199,12 @@
   "search": {
     "placeholder": "Search",
     "hint": "What are you searching for?",
-    "failed": "Nothing found"
+    "failed": "Nothing found",
+    "heading": {
+      "Post": "Posts",
+      "User": "Users",
+      "Tag": "Hashtags"
```


same as :point_up_2: ... let's add this to your other `WIP` PR, please.